manifests/init.pp

```diff
@@ -188,10 +188,12 @@
     create_resources(cron::fragment,$cron_files)
   }
 
+  # lint:ignore:undef_in_function # unsure why it get triggerd here
   if $user_crontabs_real != undef {
     validate_hash($user_crontabs_real)
     create_resources(cron::user::crontab, $user_crontabs_real)
   }
+  # lint:endignore
```

Comment:
Don't understand myself why puppet-lint is failing on this block. For me it looks mostly identically to the pattern above.